modules/ROOT/nav.adoc

```diff
@@ -20,6 +20,7 @@
   ** xref:manage-elements-visibility.adoc[Schema Element Visibility]
   ** xref:edit-elements-names.adoc[]
```

Comment:
I'm assuming it's intentional that none of these files has a nav name?

Reply:
If you leave it blank, it just uses the title of the topic